### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by leo-ri**
remove linter warnings
delete not used scripts/files/etc
add values atlasBaseURL, realmBaseURL to the Helm chart
fix samples/kubernetes/*

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -35,7 +35,4 @@ issues:
   exclude-rules:
     - path: test/
       linters:
-        - whitespace
```
Comment (vasilevp) [inline .golangci.yml:38]: why?

Comment (vasilevp) [reply .golangci.yml:38]: Makes sense - thank you

**Review by vasilevp — APPROVED** [review]

File: samples/kubernetes/atlas-service-broker-auth.yaml [hunk 2]
```diff
@@ -4,5 +4,27 @@ metadata:
   name: atlas-service-broker-auth
 type: Opaque
 stringData:
-  username: "wwkzdiwj@5ea0477597999053a5f9cbec"
-  password: 62bff0f6-ff2e-471d-9e36-6efd5fa24239
+  username: "username"
```
Comment (antonlisovenko) [inline samples/kubernetes/atlas-service-broker-auth.yaml:7]: [q] are these username/password the same as in the next secret? If so - why do we need this secret??

**Review by antonlisovenko — APPROVED** [review]
LGTM, just a couple of questions 👍

File: samples/kubernetes/atlas-service-broker-auth.yaml (hunk 2 above)
Comment (leo-ri) [reply samples/kubernetes/atlas-service-broker-auth.yaml:7]: one (first with username\password only) for the service-broker to get access for AOSB app, second(with JSON) for AOSB application/deployment itself

**Pull request merged**